packages/xdl/src/detach/Detach.js

```diff
@@ -510,17 +499,13 @@ export async function prepareDetachedBuildAsync(projectDir: string, args: any) {
   }
 }
 
-type BundleAssetsArgs = {
-  platform: 'ios' | 'android',
-  // This is the path where assets will be copied to. It should be
-  // `$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH` on iOS
-  // (see `exponent-view-template.xcodeproj/project.pbxproj` for an example)
-  // and `$buildDir/intermediates/assets/$targetPath` on Android (see
-  // `android/app/expo.gradle` for an example).
-  dest: string,
-};
-
-export async function bundleAssetsAsync(projectDir: string, args: BundleAssetsArgs) {
+// args.dest: string,
+// This is the path where assets will be copied to. It should be
+// `$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH` on iOS
+// (see `exponent-view-template.xcodeproj/project.pbxproj` for an example)
+// and `$buildDir/intermediates/assets/$targetPath` on Android (see
+// `android/app/expo.gradle` for an example).
```

Comment:
Kept this comment even though the `BundleAssetsArgs` type is gone, because it seemed helpful.